script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1113,17 +1121,18 @@ Suddenly, I hear a voice.
 [sha:24ba01e7a7b6bc4a83aa5e24fbe94c4870dd21cb]{
 -- Page 643, Offset 35118.
 -- 彼は（かたあしとかためがなくなっていたけど）まだイきていました。
-This man (although he's missing a leg and an eye) is still alive.
+// nasu really just put parentheses mid-narration
```

Comment:
just because he's a hack doesn't mean we need to be. em-dashes are parentheticals for fancy people like us
> He's still aliveーーeven though he's missing an eye and a leg.